samples/powerplay/src/main/cpp/PowerPlayMultiPlayer.cpp

```diff
@@ -109,8 +109,17 @@ void PowerPlayMultiPlayer::triggerDown(int32_t index, oboe::PerformanceMode perf
             return;
         }
     }
+
+    mSampleSources[index]->setPlayMode(performanceModeChanged);
+
     if (mAudioStream) {
-        startStream();
+        if (performanceModeChanged &&
+            mAudioStream->getPerformanceMode() == PerformanceMode::PowerSavingOffloaded &&
+            mSampleSources[index]->getCurserIndex() != 0) {
+            mAudioStream->flushFromFrame(FlushFromAccuracy::Accurate, 0);
+        } else {
+            startStream(mLastPerformanceMode);
```

Comment:
Why is mLastPerformanceMode used now? Can you make the minimum set of changes so this is easier to review?

Reply:
There was a bug where when you started a stream, it would under the hood change the performance mode back to LowLatency, this causing the Power Savings mode to be switched off when changing streams. 

This Pr wouldn't work without this change